client/src/pages/home/ModeSelectionDialog.tsx

```diff
@@ -0,0 +1,159 @@
+import {Button} from '@/components/ui/button';
+import {
+    Dialog,
+    DialogContent,
+    DialogDescription,
+    DialogFooter,
+    DialogHeader,
+    DialogTitle,
+} from '@/components/ui/dialog';
+import {Label} from '@/components/ui/label';
+import {RadioGroup, RadioGroupItem} from '@/components/ui/radio-group';
+import {Bot, Code} from 'lucide-react';
```

Comment:
Importaj ikone sa Icon suffixom (`BotIcon, CodeIcon`) - cilj je da je jasno kad se čita kod da se zna da je ikona u pitanju a ne nešto drugo